servers/cromwell/README.md

```diff
@@ -48,9 +44,42 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
 }
 ```
 
-4. Symbolically link the cromwell docker compose file depending on your `CROMWELL_URL`. For `https://cromwell.caas-dev.broadinstitute.org/api/workflows/v1`, use `cromwell-compose-caas.yaml` otherwise use `cromwell-compose-dev.yaml`, e.g:
+   **Note:** If you want to use use Job Manager against Cromwell-as-a-Service, which is using SAM/Google OAuth for authN and authZ, the `capabilities_config.json` must also include some extra fields, as well as proper scopes, which are shown as below:
```

Comment:
nit:  Google Oauth / SAM

SAM -> authz, Google Oauth -> authn  (this has them listed in reverse order)

Reply:
Oops, flipped order. Thanks for the catch, will fix the doc.